client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/BpToast.kt

```diff
@@ -0,0 +1,85 @@
+package presentation.login.composable
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import com.beepbeep.designSystem.ui.theme.Theme
+import kotlinx.coroutines.delay
+import util.BPToastDuration
+import kotlin.time.Duration
+
+@Composable
+fun BpToast(
```

Comment:
I think you mean the snackbar , you should use BPSnackBar created in design system.

Reply:
It doesn't match the design because the snackbar has elevation, and custom padding is not possible, you can test it and see the difference.

Reply:
If you mean this they are the same in design system 
![Capture](https://github.com/TheChance101/beep-beep/assets/30028905/12b19292-8a6d-44ed-ab2d-63958e1494b8)

Reply:
your image is with snackbar?

Reply:
Yes

Reply:
the problem is with BPSnackBar that doesn't match the design, but i will use the BPSnackBar and it should be fixed to match the design